datacapturegallery/src/main/java/com/google/android/fhir/datacapture/gallery/MainActivity.kt

```diff
@@ -56,6 +56,12 @@ class MainActivity : AppCompatActivity() {
                     "MyPAIN",
                     "CDS For Chronic Pain Management",
                     "cds4cpm-mypain.json"
+                ),
+                // IPRD-WITS Clinic HIV Risk assessment questionnaire
```

Comment:
Add something so people can trace to the author:

```suggestion
                // IPRD-WITS Clinic HIV Risk assessment questionnaire authored by @joiskash
```